packages/plugin-ext/src/plugin/plugin-context.ts

```diff
@@ -789,7 +789,7 @@ export function createAPIFactory(
         };
 
         return <typeof theia>{
-            version: require('../../package.json').version,
+            version: process.env['VSCODE_API_VERSION'] || require('../../package.json').version,
```

Comment:
Why is this needed? I think that's the version of the Theia plugin API rather than the VSCode version.

Reply:
Unfortunately, it is.

Reply:
You can even see this change in master of theia. Here: https://github.com/eclipse-theia/theia/blob/e33708b77117a044f5f0a3fa97309c8b9fe3664f/packages/plugin-ext-vscode/src/node/plugin-vscode-init.ts#L56

Reply:
To be honest I'm not 100% sure why Theia doesn't need it

Reply:
That being said they haven't updated to a new version of vscode-clangd yet

Reply:
Without it an incorrect VSCode version is detected as this code is used to create a default version value. It means that vscode-clangd is not loaded.

Reply:
I get a feeling that for some reason plugins are incorrectly marked and they are loaded as theia plugins instead of vscode plugins.

Reply:
Interesting, it looks like the plugin is correctly recognized as a VSCode extension by the plugin-ext-vscode package, but the version check in the vscode language server code seems to get the wrong `vscode.version`.